articles/contributing-style-guide.md

```diff
@@ -328,16 +326,16 @@ We suggest:
 
 ## Input conventions ##
 
-When a developer calls into an operation or function, the various inputs to that operation or function must be specified in a particular order, increasing the cognitive load that a developer faces in order to make use of a library.
-In particular, the task of remembering input orderings is often a distraction from the task at hand: programming an implementation of a quantum algorithm.
+When a developer calls an operation or function, input parameters must be specified in a specific order. This increases the cognitive load that a developer faces to be able to use a library.
+Remembering the order of input parameters is often a distraction from the task at hand: programming an implementation of a quantum algorithm.
 Though rich IDE support can mitigate this to a large extent, good design and adherence to common conventions can also help to minimize the cognitive load imposed by an API.
 
-Where possible, it can be helpful to reduce the number of inputs expected by an operation or function, so that the role of each input is more immediately obvious both to developers calling into that operation or function, and to developers reading that code later.
-Especially when it is not possible or reasonable to reduce the number of arguments to an operation or function, it is important to have a consistent ordering that minimizes the surprise that a user faces when predicting the order of inputs.
+Reduce the number of inputs expected by an operation or function. This way the role of each input is more immediately obvious both to developers that call the operation or function, and to developers that read the code later.
+When it is not possible or reasonable to reduce the number of arguments in an operation or function, be consistent with the order of arguments. This minimizes surprises as users predict input order.
 
-We recommend an input ordering conventions that largely derives from thinking of partial application as a generalization of currying 𝑓(𝑥, 𝑦) ≡ 𝑓(𝑥)(𝑦).
-Thus, partially applying the first arguments should result in a callable that is useful in its own right whenever that is reasonable.
-Following this principle, consider using the following order of arguments:
+We recommend an input order convention. Create this by thinking of partial application as a generalization of currying 𝑓(𝑥, 𝑦) ≡ 𝑓(𝑥)(𝑦).
+Partially apply the first arguments which results in a callable that is useful in its own right whenever that is reasonable.
```

Comment:
This changes the meaning to instruct readers to partially apply functions and operations rather than to consider the experience their users will have when they partially apply.

Reply:
I have reverted this statement.